compiler/damlc/tests/daml-test-files/Iou12.EXPECTED.rst

```diff
@@ -33,22 +35,36 @@ Templates
        - \[`Party <https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-party-57932>`_\]
        - ``regulators`` may observe any use of the ``Iou``
 
-  + **Choice Archive**
+  + **Choice** Archive
+
+    Controller\: issuer
 
     Returns\: ()
 
     (no fields)
 
-  + **Choice DoNothing**
+  + .. _type-iou12-donothing-75627:
+
+    **Choice** `DoNothing <type-iou12-donothing-75627_>`_
+
+    Controller\: let \_ \= updateAmount in
+let \_ \= this in
+let \_ \= arg in DA\.Internal\.Desugar\.toParties (owner)
```

Comment:
I'm not sure how this is going to look once rendered, we might need to escape the lambdas?

Reply:
However, this leaks our desugaring, `let _ = updateAmount in` comes from the template-let block and `let _ = this in let _ = arg in` comes from the controller expression desugaring. I'm not sure what to do with these, since they could be arbitrary expressions.